src/util/getThoughtsDEPRECATED.js

```diff
@@ -0,0 +1,36 @@
+import { store } from '../store.js'
```

Comment:
Do you know how this file got added back in? I believe this file was deleted. I think there may be some other files that were incorrectly added back in as well.